nilearn/plotting/html_surface.py

```diff
@@ -164,6 +263,27 @@ def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
         If True, image is plotted on a black background. Otherwise on a
         white background. Default=False.
 
+    bg_maps : "auto" or list of (str or numpy.ndarray), optional
```

Comment:
IIRC we agreed orally during one of the office hours that there would be no such parameter for now, or that it could only be a string corresponding to one of the fsaverage maps?